akka-actor/src/main/scala/akka/actor/CoordinatedShutdown.scala

```diff
@@ -406,6 +406,12 @@ final class CoordinatedShutdown private[akka] (
       def loop(remainingPhases: List[String]): Future[Done] = {
         remainingPhases match {
           case Nil ⇒ Future.successful(Done)
+          case phase :: remaining if !phases(phase).enabled ⇒
+            tasks.get(phase) match {
+              case null  ⇒ // This pretty much is ok as there are no tasks
+              case tasks ⇒ log.warning("Phase [{}] disabled through configuration, skipping [{}] tasks", phase, tasks.size)
```

Comment:
perhaps info is enough, they did it on purpose

Reply:
True